src/PerfView/OtherSources/Linux/LinuxPerfScriptStackSource.cs

```diff
@@ -48,29 +47,45 @@ protected override void DoInterning()
                     threadBlockedTimeAnalyzers[i] = new List<BlockedTimeAnalyzer>();
                 }
 
+                var currentCulture = Thread.CurrentThread.CurrentCulture;
```

Comment:
:memo: (For reviewers) these changes are more easily reviewed with whitespace ignored:
https://github.com/Microsoft/perfview/pull/311/files?w=1